src/api/app/views/webui2/webui/project/_submit_update_dialog.html.haml

```diff
@@ -0,0 +1,51 @@
+.modal.fade#submit-update-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'submit-update-modal-label', hidden: true } }
+  .modal-dialog.modal-dialog-centered{ role: 'document' }
+    .modal-content
+      .modal-header
+        %h5.modal-title#submit-update-modal-label Submit as Update
+      .modal-body
+        %p This will submit your project as a Maintenance Incident Request.
+
+        = form_tag(project_new_incident_request_path) do
+          = hidden_field_tag(:project, project.name)
+
+          .form-group
+            = label_tag(:sourceproject, 'Project Submitted as Update:')
+            = text_field_tag(:sourceproject, project.name, disabled: true, class: 'form-control')
+          .form-group
+            = label_tag(:releasetarget, 'Release Targets:')
+            - release_targets.each do |release_target|
+              = text_field_tag(:releasetarget, release_target, :disabled => true, class: 'form-control')
+          .form-group
+            = label_tag(:description, 'Description:')
+            = text_area_tag(:description, '', rows: 3, class: 'form-control')
+
+          .modal-footer
+            = render partial: 'webui2/shared/dialog_action_buttons'
+
+
+  -# From app/views/webui/project/_incident_request_dialog.html.erb
```

Comment:
This is a left over right?

Reply:
Yes, I used that a reference while adapting the old code to Bootstrap